graylog2-server/src/main/java/org/graylog2/rest/resources/users/UsersResource.java

```diff
@@ -295,6 +296,9 @@ public Response create(@ApiParam(name = "JSON body", value = "Must contain usern
             LOG.error(msg);
             throw new BadRequestException(msg);
         }
+        if (cr.roles() != null && cr.roles().contains(RoleServiceImpl.ADMIN_ROLENAME) && cr.isServiceAccount()) {
```

Comment:
I think this check can be circumvented by using a rolename with different capitalization. e.g. `aDMin`

Reply:
the `RoleServiceImpl` loads the role with a `toLowerCase()`